src/main/scala/viper/gobra/frontend/Parser.scala

```diff
@@ -287,7 +287,9 @@ object Parser {
       "memory", "fold", "unfold", "unfolding", "pure",
       "predicate", "old", "seq", "set", "in", "union",
       "intersection", "setminus", "subset", "mset", "option",
-      "none", "some", "get"
+      "none", "some", "get",
+      "typeOf", "isComparable",
+      "int32"
```

Comment:
Why did you introduce `int32` as a reserved keyword?

Reply:
Good question, I will remove it.